### ISSUE TITLE
No longer working with DevTest Labs

### ISSUE THREAD

**Issue by martincaddick** [Azure/azure-devtestlab#869]
For some reason this has stopped working with DevTest Labs. This has been working fine for months and in the last week or two it has stopped working :( 
<img width="885" alt="image" src="https://user-images.githubusercontent.com/73564857/236884071-b7948ab8-c4fa-44c4-b6a9-1592ef1a1198.png">

**Comment by Mike-Honey** [comment]
You probably needed to be more specific than "this". The repo has thousands of files.

**Comment by martincaddick** [comment]
You're right @Mike-Honey I probably should have added more info. At some point over the last 2 years it started working again _shrug_
